SConstruct

```diff
@@ -956,16 +956,16 @@ if selected_platform in platform_list:
         env.vs_incs = []
         env.vs_srcs = []
 
-    if env["compiledb"]:
-        # Generating the compilation DB (`compile_commands.json`) requires SCons 4.0.0 or later.
-        from SCons import __version__ as scons_raw_version
-
-        scons_ver = env._get_major_minor_revision(scons_raw_version)
-
-        if scons_ver < (4, 0, 0):
-            print("The `compiledb=yes` option requires SCons 4.0 or later, but your version is %s." % scons_raw_version)
-            Exit(255)
+    # CompileDB
+    # SCons
```

Comment:
Not sure what the `# SCons` comment conveys here, I would just remove this one and keep only `# CompileDB`.

Reply:
A leftover!